apps/codeforafrica/src/lib/data/utils/posts.js

```diff
@@ -0,0 +1,180 @@
+import formatDate from "@/codeforafrica/utils/formatDate";
+
+export function formatTags(tags) {
+  const excludedTags = ["stories", "opportunities"];
+
+  const filteredTags = tags.filter(
+    (tag) => !excludedTags.includes(tag.name.toLowerCase()),
+  );
+
+  const distinctTags = [
+    ...new Set(filteredTags.map((tag) => tag.name.toLowerCase())),
+  ];
+
+  const sortedTags = distinctTags.sort().map((tagName) => {
+    return filteredTags.find((tag) => tag.name.toLowerCase() === tagName);
+  });
```

Comment:
```suggestion
  const filteredTags = tags.filter(
    (tag) => !excludedTags.includes(tag.name.toLowerCase()),
  );
  const distinctTags = [
    ...new Set(filteredTags.map((tag) => tag.name.toLowerCase())),
  ];
  const sortedTags = distinctTags.sort().map((tagName) => {
    return filteredTags.find((tag) => tag.name.toLowerCase() === tagName);
  });
  
```

1. Why so many empty lines between lines of code?
2. Can you also describe what are we really trying to do here? I see we're also sorting tags in the UI so what does this sort add?

Reply:
@kilemensi The initial plan was to remove duplicates, filter out the primary Tags and sort them. I think I can update it to remove duplicates and sort them, then remove the primary Tag in the UI